source/tyr/tyr/api.py

```diff
@@ -118,6 +118,14 @@
     '/v1/equipments_providers/<string:id>',
 )
 
+api.add_resource(
+    ridesharing_service.RidesharingService,
+    '/v0/ridesharing_services',
+    '/v0/ridesharing_services/<string:id>',
+    '/v1/ridesharing_services',
+    '/v1/ridesharing_services/<string:id>',
```

Comment:
Any reason why you need both  `v0` and  `v1` ? I don't see any difference in the implementation between the two.

Reply:
We always manage the two versions v0 and v1.
This is what we do for the "StreetNetworkBackend" API, ...

V0 and V1: it's exactly the same implementation

Reply:
Even though it was done that way, it doesn't mean it was done the right way ;)
if those 2 endpoints have the exact same implementation, it would be less confusing to implement the version handling the same way other enpoints do : 
```python
 api.add_resource(
    ridesharing_service.RidesharingService,
    '/v<int:version>/ridesharing_services',
    '/v<int:version>/ridesharing_services/<string:id>',
)
```